src/docs/product/integrations/gcp-cloud-run/index.mdx

```diff
@@ -6,8 +6,16 @@ description: "Learn more about Sentry's Google Cloud Run integration and how it
 
 Cloud Run helps developers save time in building and deploying their applications. Sentry helps save time in resolving production issues by providing detailed debugging capabilities.
 
+This integration needs to be set up in each project for which you wish to use it. It is maintained and supported by the [Sentry community](https://forum.sentry.io/).
```

Comment:
@ajjindal will you confirm if this project level integration is maintained by the Sentry community?

Reply:
The Google Cloud Run integrations page is more of a place holder for us to be able to partner with GCP. So, we should skip this addition.